src/formatDate.js

```diff
@@ -8,7 +8,39 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[fromFormat.length - 1];
+  const toSeparator = toFormat[toFormat.length - 1];
+
+  const newFormat = fromFormat.slice(0, -1);
+  const newForm = toFormat.slice(0, -1);
```

Comment:
The variable `newForm` should be named more descriptively, such as `newToFormat`, to better reflect its purpose and maintain consistency with `newFormat`.